css/logo-nav.css

```diff
@@ -289,23 +453,24 @@ a {
     -webkit-box-sizing: border-box;
     box-sizing: border-box;
 }
+
+.companies .col img{
+    display: block;
+    width: 100%
+}
 @media(max-width:1200px) {
     .companies li{
         width:23%;
     }
 }
-@media(max-width:767px) {
+@media(max-width:768px) {
```

Comment:
Thanks for the catch. That media query has been changed back to 767px. Also, changed all media queries that have max-width to 767px and min-width to 768px.